nservicebus/upgrades/externalize-encryption.md

```diff
@@ -9,14 +9,21 @@ related:
  - nservicebus/security/property-encryption
 ---
 
-The [Message Property Encryption feature](/nservicebus/security/property-encryption.md), has been moved from the NServiceBus core to the separate NuGet package [NServiceBus.Encryption.MessageProperty](https://www.nuget.org/packages/NServiceBus.Encryption.MessageProperty/). That package should be used in order to encryption message properties when using versions NServiceBus Versions 6.2 and above.
-
+The [Message Property Encryption feature](/nservicebus/security/property-encryption.md), has been moved from the NServiceBus package to the separate [NServiceBus.Encryption.MessageProperty](https://www.nuget.org/packages/NServiceBus.Encryption.MessageProperty/) NuGet package. That package should be used to encrypt message properties when using versions NServiceBus Versions 6.2 and above.
 
 The API was also modified.
 
 ## Removed APIs
 
-Configuring encryption [Via app.config](/nservicebus/security/property-encryption.md#configuration-via-app-config) and [Via IProvideConfiguration](/nservicebus/security/property-encryption.md#configuration-via-iprovideconfiguration) have been removed. Instead use [Configuration via code](/nservicebus/security/property-encryption.md#configuration-via-code).
+Configuring encryption [via app.config](/nservicebus/security/property-encryption.md#configuration-via-app-config) and [via IProvideConfiguration](/nservicebus/security/property-encryption.md#configuration-via-iprovideconfiguration) have been removed. Instead use [configuration via code](/nservicebus/security/property-encryption.md#configuration-via-code).
+
+
+## Migration steps
+
+When migrating endpoints to use the `NServiceBus.Encryption.MessageProperty` package:
```

Comment:
@timbussmann Should we make it clear that the user needs to add the nuget package, update the configuration, and deploy the updated endpoints as part of step 1?